.github/workflows/differential-shellcheck.yaml

```diff
@@ -0,0 +1,32 @@
+---
+# https://github.com/redhat-plumbers-in-action/differential-shellcheck#readme
+
+name: Differential ShellCheck
+on:
+  push:
+    branches:
+      - master
```

Comment:
@tchaikov I know this one is already queued, @Annamikhlin raised a good point, why do you need to run it during the push to `master` if you are running during the PR opening?

Reply:
@yaronkaikov this is indeed a good question. i pondered on it for a while. i think it depends on the development model: we merge pull requests into master in batch. so there are chances that two pull requests are sane if we look at them individually, but their combination could cause a bug.  if we trigger this workflow with push to master, we would be able to catch this issue.

what do you think?

Reply:
I assume it;s an end case, but if it will cover it, i would leave it as is